website/docs/systemversion.md

```diff
@@ -44,15 +44,21 @@ filter "system:macosx"
 
 ### Apple Targets ###
 
-Under macOS this sets the minimum version of the operating system required for the app to run and  is equivalent to setting the `-macosx-version-min` (or newer `-macos-version-min`) compiler flag.
+Under macOS this sets the minimum version of the operating system required for the app to run and is equivalent to setting the `-mmacosx-version-min` (or newer `-mmacos-version-min`) compiler flag.
 
-The same is true for iOS, iPadOS, tvOS, and watchOS system targets except it is equivalent to setting the `-miphoneos-version-min` (or newer `ios-version-min`) compiler flag.
+The same is true for iOS, iPadOS, and watchOS system targets except it is equivalent to setting the `-miphoneos-version-min` (or newer `-mios-version-min`) compiler flag.
```

Comment:
This is a pre-existing issue, and not directly relevant to this PR so I haven't touched it, but I want to say that mentioning `watchOS` in this sentence is not super correct in my opinion.

First, Premake doesn't directly support watchOS, so mentioning it at all without also saying Premake doesn't support it seems wrong.

Second, watchOS doesn't even use this compiler flag as Clang has a separate `-mwatchos-version-min` compiler flag just for watchOS, unless Clang has some weird internal logic where it falls back to using the iOS flag when the watchOS flag isn't specified